test/matrix.cc

```diff
@@ -231,7 +231,7 @@ TEST(Matrix, test_matrix) {
 
   CostMatrix cost_matrix_abort_source;
   cost_matrix_abort_source.SourceToTarget(request, reader, mode_costing, sif::TravelMode::kDrive,
-                                          90000.0);
+                                          7000.0);
```

Comment:
this here was testing that we don't find certain connections based on the max matrix distance. since we changed the concept to distance now, it seems we need to lower that significantly to have a similar effect.